src/services/MetricsCalculation.js

```diff
@@ -0,0 +1,30 @@
+import * as util from '../api/util';
+
+const COMPLETION_RATE_ACCURACY = 0.1; // percent in decimal format, must not be null.
+
+function isVideoTimeSegmentInChunk(videotime_start, videotime_end, chunktime_start, chunktime_end) {
+  if (videotime_end - videotime_start === 0) { return false; }
+  if (videotime_end < chunktime_start || videotime_start > chunktime_end) {
```

Comment:
this can be reduced to `return !(videotime_end < chunktime_start || videotime_start > chunktime_end)`